WireUI/Sources/WireSidebarUI/Views/SidebarMenuItemView.swift

```diff
@@ -50,16 +50,16 @@ struct SidebarMenuItemView: View {
     // MARK: -
 
     var body: some View {
-        let accentColor = wireAccentColorMapping.uiColor(for: wireAccentColor)
+        let accentColor = wireAccentColorMapping?.uiColor(for: wireAccentColor) ?? .systemGray
```

Comment:
why do we have this variable in body? can't we resolve it in init

Reply:
fixed: f41bedb14c990c99e1f03144e449f010c2a2def7